### ISSUE TITLE
its been 2 years, my (ti)boy

### ISSUE THREAD

**Issue by teddblue** [calc84maniac/tiboyce#145]
just wondering when I should expect another update (and how to update it). That's all, bye
PS: I know how hard coding can be and I'm not rushing anyone in any way. just curious about the future of this project and what we could expect in the months to come

**Comment by adriweb** [comment]
Don't worry, the dev branch is quite active, especially these days, with new features, fixes, and improvements are almost done daily. But as mentioned in https://github.com/calc84maniac/tiboyce/issues/136#issuecomment-916184378 it's not ready just yet for a release (but soon, apparently). You can try a pre-release from the page I just linked.

**Comment by calc84maniac** [comment]
I should note that the file I uploaded for that linked issue is not a full pre-release, just a targeted fix for a hardware initialization bug. It doesn't contain any new emulator functionality.

That said yes, I very much want to make a new release soon-ish (definitely want to target before the end of the year at least). I've been making a ton of progress with compatibility and performance improvements, and last I checked I've fixed all of the open issues for broken games.

After that I'll have a clean break to start massive changes to the emulator to attempt Game Boy Color support.
